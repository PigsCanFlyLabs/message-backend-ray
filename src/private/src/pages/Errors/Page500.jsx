import React from 'react';
import { Link } from 'react-router-dom';

export default function Page500() {
  return (
    <div className="w-100 h-100 d-flex align-items-center justify-content-center flex-direction-column">
      <h3 className="sf-text-color-primary"> 500 | Internal Server Error</h3>
      <h4>
        <Link to="/">Go to main page</Link>
      </h4>
    </div>
  );
}
