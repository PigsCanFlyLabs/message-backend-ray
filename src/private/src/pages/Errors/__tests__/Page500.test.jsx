import React from 'react';

import Page500 from '../Page500';

describe('Page500', () => {
  it('should render without errors', () => {
    const component = <Page500 />;
    expect(component).toMatchSnapshot();
  });
});
