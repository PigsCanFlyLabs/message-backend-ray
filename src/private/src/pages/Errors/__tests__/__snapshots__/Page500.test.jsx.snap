// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page500 should render without errors 1`] = `<Page500 />`;
